hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/entity/sql/SqlEntityListener.java

```diff
@@ -214,6 +225,7 @@ private void executeBatches() {
             fileDataPgCopy.copy(fileData, connection);
             liveHashPgCopy.copy(liveHashes, connection);
             nonFeeTransferPgCopy.copy(nonFeeTransfers, connection);
+            nftTransferPgCopy.copy(nftTransfers, connection);
```

Comment:
I thought we were gonna move this and all the transfers to the end to be the last copies.
That way all entities noted in the transfers would already be present in DB?

Reply:
Something like
```suggestion
            persistNfts();
            ...
            nftTransferPgCopy.copy(nftTransfers, connection);
```

Which would call
```
private void persistNfts() {
        Stopwatch stopwatch = Stopwatch.createStarted();
        nftRepositorysaveAll(nfts);
        log.info("Inserted {} nfts in {}", nfts.size(), stopwatch);
}
```

Reply:
I realize the save occurs in EntityRecordListener instead so the placement here doesn't really matter